proposals/0064-transaction-receipt.md

```diff
@@ -0,0 +1,416 @@
+---
+simd: '0064'
+title: Transaction Receipts
+authors:
+  - Anoushk Kharangate (Tinydancer)
+  - Harsh Patel (Tinydancer)
+  - Richard Patel (Jump)
+category: Standard
+type: Core
+status: Draft
+created: 2023-06-20
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This proposal introduces two new concepts to the Solana runtime.
+
+- TransactionReceiptData, a deterministic encoding of state changes induced by
+  a transaction;
+- Transaction Receipt Tree, a commitment scheme over all transaction receipts
+  in a block.
```

Comment:
```suggestion
Here we propose a mechanism for proving transaction inclusion into a block in
the Solana protocol. This is a pre-requisite for several use-cases that would
like to build upon a [Simple Payment Verification](https://en.wikipedia.org/wiki/Bitcoin_network#Payment_verification) like construction.

We employ the well-known [Merkle Tree](https://en.wikipedia.org/wiki/Merkle_tree)
data structure to compress a block's transactions and their results into a compact
identifier, with which inclusion proofs can be generated
```

the summary should (MUST!) have a little more color than copypastaing the terminology section :wink:

Reply:
We were trying to stick to the format but we will keep this mind going forward, thank you.